pkg/snapshotter/snapshotter_csi.go

```diff
@@ -1163,7 +1163,25 @@ func buildJobSpec(
 	}, " ")
 
 	jobPodBackOffLimit := int32(1)
-
+	storkPodNs, err := k8sutils.GetStorkPodNamespace()
```

Comment:
need to have a check for storkPodNs too. it can be empty string.

Reply:
Added a check in GetStorkPodNamespace() function itself to return err, if the ns is empty.